src/components/Verse/AdvancedCopy/SelectorContainer.tsx

```diff
@@ -38,6 +38,7 @@ const SelectorContainer: React.FC<SelectorProps> = ({ type, value, dropdownItems
     <Combobox
       id={type}
       value={value}
+      clearable={false}
```

Comment:
tested this, and it seems that clearable allows clearning (?) by typing back space. Out of scope of this PR, but fyi.